Source/Fuse.Scripting.JavaScript/ThreadWorker.uno

```diff
@@ -133,12 +133,11 @@ namespace Fuse.Scripting.JavaScript
 						_exceptionQueue.Enqueue(e);
 					}
 				}
-				else
-					_idle.Set();
 
 				try
 				{
-					_fuseJS.UpdateModules(_context);
+					var activity = _fuseJS.UpdateModules(_context);
+					didAnything ||= activity;
```

Comment:
[C# doesn't support `||=`](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/), so if Uno supports this, I'd consider that a compiler-bug. Please don't use it.